skops/io/_general.py

```diff
@@ -4,19 +4,13 @@
 
 import numpy as np
 
-from ._utils import (
-    _import_obj,
-    get_instance,
-    get_state,
-    try_get_instance,
-    try_get_state,
-)
+from ._utils import _import_obj, get_module, try_get_instance, try_get_state
 
 
 def dict_get_state(obj, dst):
     res = {
         "__class__": obj.__class__.__name__,
-        "__module__": inspect.getmodule(type(obj)).__name__,
+        "__module__": get_module(obj),
```

Comment:
since at least in one occasion, and probably other places, we get the module of `obj` instead of `type(obj)`, what do you think of doing `get_module(type(obj))`? It also reads more explicit to me.